vendor/modules.txt

```diff
@@ -393,6 +393,10 @@ github.com/grpc-ecosystem/go-grpc-prometheus
 github.com/grpc-ecosystem/grpc-gateway/internal
 github.com/grpc-ecosystem/grpc-gateway/runtime
 github.com/grpc-ecosystem/grpc-gateway/utilities
+# github.com/hashicorp/errwrap v1.0.0
+## explicit
+# github.com/hashicorp/go-multierror v1.0.0
+## explicit
```

Comment:
Right, I only ran `make revendor` once.
Currently, it is not idempotent because `go mod vendor` is called before `go mod tidy`. I inverted the order to make it idempotent in a new commit :)